initial_settings/initial.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python -tt
+
+import subprocess
+from sys import argv
+import dnf
+
+
+class DnfUpgrageTo():
+    def create_repo(self):
```

Comment:
Unfortunately it is needed for dnf_version() function. It search for exact name of dnf package for upgrade-to command.